### DISCUSSION TITLE
多个子图如何按照自己想要的方式插入？

### DISCUSSION THREAD

**Discussion by chenhongyang113** [sjtug/SJTUThesis#812] [提问] [answered]
问题如下：
比如我现在有下面这个图中分别的六个图，可以用什么办法按照如图的形式插入到正文呢？模板里自带的好像是只有并排的两个图的情况，这六个图不需要单独的（a）（b）这样的题注，就一个总的标题就可以，自己试过好像不能用subfigure，不知道应该怎么办。谢谢各位大神指导！
<img width="662" alt="2022-05-23_0-17-29" src="https://user-images.githubusercontent.com/30236051/169705362-aa377a72-be79-487d-abc8-1a94f056b445.png">

**Comment by LogCreative** [comment, marked as answer]
直接连续插入图片即可，控制图片宽度。

```latex
\begin{figure}[h]
    \centering
    \includegraphics[width=0.45\textwidth]{example-image-plain}
    \includegraphics[width=0.45\textwidth]{example-image-plain}
    \includegraphics[width=0.45\textwidth]{example-image-plain}
    \includegraphics[width=0.45\textwidth]{example-image-plain}
    \includegraphics[width=0.45\textwidth]{example-image-plain}
    \includegraphics[width=0.45\textwidth]{example-image-plain}
    \caption{总标题}
    \label{fig:myfig}
\end{figure}
```
<img width="327" alt="image" src="https://user-images.githubusercontent.com/61653082/169705826-4ca08539-e0a1-4c8e-ad08-f62ebff9b91e.png">

**Reply by chenhongyang113** [reply]
谢谢您及时回复！！
之前傻了，忘了可以这样直接插入图片了
